src/main/kotlin/org/rust/lang/core/completion/RsAttributeCompletionProvider.kt

```diff
@@ -84,15 +80,9 @@ object RsAttributeCompletionProvider : RsCompletionProvider() {
 
     override val elementPattern: ElementPattern<PsiElement>
         get() {
-            val outerAttrElem = psiElement<RsOuterAttr>()
-            val innerAttrElem = psiElement<RsInnerAttr>()
-            val metaItemElem = psiElement<RsMetaItem>().and(
-                psiElement().withParent(outerAttrElem) or psiElement().withParent(innerAttrElem)
-            )
-
             return psiElement()
                 .withLanguage(RsLanguage)
-                .withParent(psiElement<RsPath>().withParent(metaItemElem))
+                .withParent(psiElement<RsPath>().withParent(rootMetaItem))
```

Comment:
Let's (try to) replace the pattern to
```
psiElement(RsElementTypes.IDENTIFIER).withParent(psiElement<RsPath>().withParent(rootMetaItem))
```
just because it's more explicit and restrictive